client/src/helpers/APICalls/login.ts

```diff
@@ -16,11 +16,10 @@ export const login = async (email: string, password: string): Promise<AuthApiDat
 };
 
 export const loginWithDemo = async (): Promise<AuthApiData> => {
-  console.log('Its working');
-
   const fetchOptions: FetchOptions = {
     method: 'POST',
     headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ email: 'DemoUser@gmail.com', password: 'DemoPassword' }),
```

Comment:
The whole purpose of wanting to do the demo auth on the back-end was to not expose the credentials on the client side. Please remove this as we do that on the back-end.